.github/workflows/main.yml

```diff
@@ -40,7 +46,7 @@ jobs:
         context: ./
         file: ./Dockerfile
         push: true
-        tags: ${{ secrets.DOCKER_HUB_USERNAME }}/drivers-test:latest.${{ steps.versions.outputs.TAG }}
+        tags: ${{ secrets.DOCKER_HUB_USERNAME }}/drivers-test/${{ matrix.crystal }}:${{ steps.versions.outputs.TAG }}
```

Comment:
the `steps.versions.output` here is pretty tightly coupled to our platform release process.